build/build.generic.cmd

```diff
@@ -0,0 +1,38 @@
+@echo off
+
+rem
+rem Parameters
+rem   %1: vs_solution_version: VS solution version: VS2012, VS2013, VS2015
+rem   %2: vs_configuration:    VS configuration:    Debug, Release
+rem   %3: vs_target:           target:              Build or Rebuild
+rem   %4: vs_platform:         platform:            x64 or Win32
+rem   %5: vs_toolset:          toolset:             v100, v110, v120, v140
+rem
+
+set vs_solution_version=%1
+set vs_configuration=%2
+set vs_target=%3
+set vs_platform=%4
+set vs_toolset=%5
+
+set msbuild="%windir%\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe"
```

Comment:
I copied and adapted what was done in GitExtensions.

I'm not exactly sure how I could read a value from the registry and insert it into a variable, from a `.cmd` script...